modules/remotecontrol/Receiver.js

```diff
@@ -25,13 +31,24 @@ export default class Receiver extends RemoteControlParticipant {
             = this._onRemoteControlEvent.bind(this);
         this._userLeftListener = this._onUserLeft.bind(this);
         this._hangupListener = this._onHangup.bind(this);
+        // We expect here that even if we receive the supported event earlier
+        // it will be cached and we'll receive it.
+        transport.on('event', data => {
+            if(data.name === REMOTE_CONTROL_EVENT_TYPE) {
```

Comment:
There are a lot other places where this is called event. I think after @lyubomir 's commit this is fixed, right?